app/src/main/java/com/project/technion/appark/activities/SearchParkingsActivity.java

```diff
@@ -96,15 +106,18 @@ public void onDataChange(@NonNull DataSnapshot dataSnapshot) {
                     if(gc.isPresent()) {
                         List<Address> list = null;
                         try {
-                            list = gc.getFromLocationName(address, 1);
+                            if(address != "" && address != null)
+                                list = gc.getFromLocationName(address, 1);
                         } catch (IOException e) {
-                            e.printStackTrace();
+                            //e.printStackTrace();
```

Comment:
in case of an exception, do we want the app keep running as usual?

Reply:
there's no point to make the app crash. An error their tells us that something went wrong with the location services or that the searched address was not found.